### ISSUE TITLE
Provide a mechanism for an extension to invoke Undo on an editor without making it active

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#18142]
tl;dr: I want something like `TextEditor.editor.undo()` added.

I am writing an extension that takes an existing text editor and opens a paired Webview to visually edit the text. (Text is XML representing a state chart in SCXML; webview is a graphical representation of the states and transitions.) Changes in the text editor cause the webview to update; changes in the webview cause the text editor to update (currently via wholesale text replacement using `TextEditor.edit()`).

![image](https://user-images.githubusercontent.com/14778/221060877-8d01948d-f0c7-4086-aed3-457916f2ea73.png)

If the user makes a mistake when editing the webview, I want them to be able to Undo via `ctrl-z`. If I invoke `vscode.commands.executeCommand('undo')` with the Webview active, nothing happens. They can click in the text editor and hit `ctrl-z` there, which works perfectly, but it's a cumbersome workflow.

If an extension is allowed to rewrite the text in a text editor, that extension should be able to Undo these changes by sending an undo command to the same editor, even if it's not focused.

As a workaround, I would even accept the extension being allowed to activated a visible text editor, if that meant I could send the command and then bring focus back to my webview.
